src/app/modules/rup/components/elementos/rupers/seccionado/seccion.component.html

```diff
@@ -15,14 +15,17 @@
             </ng-container>
 
         </div>
-        <div [class]="colForText" *ngIf="!(conceptosSeleccionar?.length > 0) && params.showText">
-            <textarea *ngIf="!soloValores" class="form-control" [class.svgTextarea]="!registro?.valor"
-                      [(ngModel)]="registro.valor" rows="6" name="descripcion" [required]="params.textRequired">
-            </textarea>
-            <p *ngIf="params.showText && soloValores" class="readonly mb-5">
-                {{registro.valor}}
-            </p>
-        </div>
+        <ng-container *ngIf="!(conceptosSeleccionar?.length > 0) && params.showText">
```

Comment:
```suggestion
        <div  [class]="colForText" *ngIf="!(conceptosSeleccionar?.length > 0) && params.showText">
```

Esto tiene que seguir siendo un div con las clases que corresponde, sino se pierde la funcionalidad de orientación de los componentes.

Reply:
@liquid36  corregido